src/main/resources/interface-specification.yml

```diff
@@ -2768,16 +2768,6 @@ paths:
           description: Query to filter EServices by name
           schema:
             type: string
-        - in: query
-          name: consumersIds
```

Comment:
I still don't remember why this parameter has been added, I don't find it in the initial requirement, but are we sure it is not used by the frontend?

Reply:
`consumerIds` is actually used.
I need to change the PR.